portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,376 @@
+  background-color: #c7e8ff;
+  border-radius: 10px;
+  transition: .2s;  
+}
+
+.card button:hover {
+  background-color: #d5eeff;
+}
+
+/*Image Class*/
+.propic {
+  width: 150px;
+  border:2px solid aliceblue;
+  border-radius:50%;
+  margin: 10px;
+}
+
+/*Columns*/
+.column {
+  float: left;
+  padding: 20px;
+}
+
+.quarter {
+  width: 25%;
+}
+
+.half {
+  width: 50%;
+}
+
+.third {
+  width: 33.33%;
+}
+
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+/*Navigation Bar*/
+#nav{
+  background: rgba(12, 12, 12, 0.719);
+  z-index: 9999;
+  padding: 8px 15px 0px 15px;
+}
+
+#nav ul {
+  list-style-type: none;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+}
+
+#nav li {
+  float: right;
+}
+
+#nav li a {
+  color: #fcfafc;
+  display: block;
+  text-align: center;
+  padding: 14px 25px;
+  text-decoration: none;
+}
+
+#nav li a:hover {
+  color: rgb(197, 197, 197);
+}
+
+.sticky {
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+/*TIMELINE CSS (taken from W3School Tutorial and edited)*/
+/* The actual timeline (the vertical ruler) */
+.timeline {
+  position: relative;
+  max-width: 1200px;
+  margin: 70px auto;
+}
+
+/* The actual timeline (the vertical ruler) */
+.timeline::after {
+  content: '';
+  position: absolute;
+  width: 6px;
+  background-color: white;
+  top: 0;
+  bottom: 0;
+  left: 50%;
+  margin-left: -3px;
+}
+
+/* Container around content */
+.tl-container {
+  padding: 10px 40px;
+  position: relative;
+  background-color: inherit;
+  width: 50%;
+  text-align: left;
+}
+
+/* The circles on the timeline */
+.tl-container::after {
+  content: '';
+  position: absolute;
+  width: 25px;
+  height: 25px;
+  right: -17px;
+  background-color: white;
+  border: 4px solid #FF9F55;
+  top: 15px;
+  border-radius: 50%;
+  z-index: 1;
+}
+
+/* Place the container to the left */
+.left {
+  left: 0;
+}
+
+/* Place the container to the right */
+.right {
+  left: 50%;
+}
+
+/* Add arrows to the left container (pointing right) */
+.left::before {
+  content: " ";
+  height: 0;
+  position: absolute;
+  top: 22px;
+  width: 0;
+  z-index: 1;
+  right: 30px;
+  border: medium solid white;
+  border-width: 10px 0 10px 10px;
+  border-color: transparent transparent transparent white;
+}
+
+/* Add arrows to the right container (pointing left) */
+.right::before {
+  content: " ";
+  height: 0;
+  position: absolute;
+  top: 22px;
+  width: 0;
+  z-index: 1;
+  left: 30px;
+  border: medium solid white;
+  border-width: 10px 10px 10px 0;
+  border-color: transparent white transparent transparent;
+}
+
+/* Fix the circle for containers on the right side */
+.right::after {
+  left: -16px;
+} 
+
+/* The actual content */
+.tl-content {
+  padding: 20px 30px;
+  background-color: white;
+  position: relative;
+  border-radius: 6px;
+  color: #474e5d
+}
+
+.tl-content:hover {
+  color: #8a94aa
+}
+
+/* Media queries - Responsive timeline on screens less than 600px wide */
+@media screen and (max-width: 600px) {
```

Comment:
Optional: If you wanted to try out some responsive styling, I would suggest adding
`<meta name="viewport" content="width=device-width, initial-scale=1">` to the HTML because otherwise the text on the page will be [really small](https://imgur.com/a/Jny4ZI9) on mobile. Then you would just need a few more style tweaks to make things look right on mobile.

Reply:
okay, ty, added!